Lesson9/anna/Lesson9-aysel/styles.css

```diff
@@ -0,0 +1,214 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  display: grid;
+  margin: 100px auto;
+  grid-gap: 10px 10px;
+  grid-template-columns: 2fr 4fr 11fr 4fr 2fr;
+}
+img {
+  max-width: 100%;
+}
+
+.post > * {
+  grid-column: 3 / span 1;
+}
+.clock,
+.youtube-pic {
+  width: 100%;
+  grid-column: 2 / span 3;
+}
+
+h1 {
+  font-size: 42px;
+  font-weight: 700;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 52px;
+  letter-spacing: -0.011em;
+}
+
+.first-pic {
+  border-radius: 50px;
+  position: relative;
+}
+.second-pic {
+  position: relative;
+  margin-left: -14px; /*i couldnt find any other way to put this pic on the first one*/
+  border-radius: 50px;
+  border: 2px solid rgb(255, 255, 255);
+}
+
+.publisher {
```

Comment:
I think there should be a flex/grid container holding 2 small images (inside their own div) and other text elements. This way it will be a lot easier to adjust layout.